### PR TITLE
Fix onBlocklyBlock & refactor drawing block code & change block content

### PR DESCRIPTION
**Pull request by AmbientOSX**
@jyucra @BrettASwift @kadst43 

Removes some log code that was left in as well...

### CODE REVIEW

**Comment by kadst43** [comment]
@BrettASwift: Going to leave this one for you...

File: source/view/blocks.js [hunk 1]
```diff
@@ -1673,12 +1675,12 @@ Blockly.JavaScript['draw_triangle'] = function(block) {
 
   var moveC = constructBlockExeFuncCall( block, actionC, 'moveRadial' );
 
-  var start = "vwf.fireEvent( '" + vwf_view.kernel.application() + 
-                  "', 'blockExecuted', " + " [ 'startTriangle', '" + block.id + "', '" + block.data + "', " + 1 + " ] );\n";
-  var end = "vwf.fireEvent( '" + vwf_view.kernel.application() + 
-                  "', 'blockExecuted', " + " [ 'endTriangle', '" + block.id + "', '" + block.data + "', " + 1 + " ] );\n";
-  var mark = "vwf.fireEvent( '" + vwf_view.kernel.application() + 
-                  "', 'blockExecuted', " + " [ 'markPoint', '" + block.id + "', '" + block.data + "', " + 1 + " ] );\n";
+  var start = "vwf.callMethod( '" + vwf_view.kernel.application() +
```
Comment (BrettASwift) [inline source/view/blocks.js]: What context is this called in? We should not be accessing `vwf` directly. Would `this` be the blockly driver? Can we use `vwf_view.kernel` instead?

Comment (AmbientOSX) [inline source/view/blocks.js]: Yes, this is the Blockly driver. I have tried using vwf_view.kernel with mixed success in the past. This seems to guarantee that the method is called or event fired. I am not sure why.

Comment (BrettASwift) [inline source/view/blocks.js]: Are you able to use `this.kernel.callMethod` then? Have you tried that?

Comment (AmbientOSX) [inline source/view/blocks.js]: I haven't tried that yet, I'll give it a shot

Comment (AmbientOSX) [inline source/view/blocks.js]: Yeah... neither vwf.kernel, nor vwf_view.kernel, nor this.kernel work with the current configuration of the blockly driver. Im wondering if there is a quick change that could be made to set the right scope... maybe relating to this line ''' interpreter.setProperty( scope, 'vwf', myVwf ) ''' at line 507 of blockly.js model-side?

**Comment by AmbientOSX** [comment]
Unless there are objections I am going to go ahead and merge this @BrettASwift - we can take a look at replacing the direct vwf call next week?

**Comment by BrettASwift** [comment]
Yeah, that's fine.

**Pull request merged**